### ISSUE TITLE
about use Markdown and album both

### ISSUE THREAD

**Issue by zitn** [tucnak/telebot#205]
when I use markdown and album like this:
`var album tb.Album`
`album = append(album, &tb.Photo{File: tb.FromURL(image1), Caption: "test"})`
`album = append(album, &tb.Photo{File: tb.FromURL(image2)})`
`bot.SendAlbum(&tb.Chat{ID: "123456"}, album, &tb.SendOptions{ParseMode: tb.ModeMarkdown})`
I don't know if this is my fault or a bug. If it is my fault, please tell me the correct usage. Anyway, thanks for your time.

**Comment by Kohinor46** [comment]
Hi!
Do you find way to do it? Can you please tell it?

**Comment by demget** [comment]
@Kohinor46 You need to send album with markdown?

**Comment by ailinykh** [comment]
Got the same issue but [sendMediaGroup](https://core.telegram.org/bots/api#sendmediagroup) does not support parse_mode yet

**Comment by demget** [comment]
@ailinykh Set a `ParseMode` field inside the first photo, which should has a caption.

**Comment by ailinykh** [comment]
@demget  `ParseMode` applied to `SendMessage` method not to [Photo](https://github.com/tucnak/telebot/blob/c2cb2be62313ead3050ced5184fb086438106920/media.go#L22) or [Video](https://github.com/tucnak/telebot/blob/c2cb2be62313ead3050ced5184fb086438106920/media.go#L114)

**Comment by demget** [comment]
In this case, if you want to apply a parse mode, you need to set it for the first photo. I agree that this behaviour should be changed to use `SendOptions`. Follow `v3` for further updates.

**Comment by ailinykh** [comment]
@demget can you provide an example please?

**Comment by demget** [comment]
```go
b.SendAlbum(m.Sender, tb.Album{
  &tb.Photo{File: ..., Caption: "*text*", ParseMode: tb.ModeMarkdown},
  &tb.Photo{File: ...},
})
```

**Comment by ailinykh** [comment]
`unknown field ParseMode in struct`

**Comment by ailinykh** [comment]
Probably I should use telebot v3 for that.
Thanks.

**Comment by demget** [comment]
Telebot v3 is still under development, although it's working, use it on your risk.

> unknown field ParseMode in struct

Make sure you're using the latest `v2.3` version.

**Comment by demget** [comment]
Fixed in `v2.4` and will be merged into `v3` soon.
